msteams-platform/messaging-extensions/design/messaging-extension-design.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: Designing your message extension
-description: Learn how to design a Teams message extension and get the Microsoft Teams UI Kit. Describes teams design guidelines reference message extensions tips best practice
+description: Learn how to design a Teams message extension and get the Microsoft Teams UI Kit. Describes about Teams design guidelines reference message extensions tips and best practice.
```

Comment:
Just check once if the article's focus is about msg. extn. vs. getting the UI kit. Right now, the metadata suggests equal weightage for both. If you think it is OK, then consider mentioning the UI kit in the passing. Eg. design msg extn. using the kit.

Reply:
edited and modified the description